spec/features/hiring_staff_can_add_stats_to_a_vacancy_spec.rb

```diff
@@ -0,0 +1,140 @@
+require 'rails_helper'
+require 'sanitize'
+RSpec.feature 'Submitting effectiveness feedback on expired vacancies', js: true do
+  NOTIFICATION_BADGE_SELECTOR = "[data-test='expired-vacancies-with-feedback-outstanding']".freeze
+  JOB_TITLE_LINK_SELECTOR = '#job-title.view-vacancy-link'.freeze
+
+  let(:school) { create(:school) }
+  before(:each) do
+    stub_hiring_staff_auth(urn: school.urn)
+    build(:vacancy, :published_slugged, school: school)
```

Comment:
Unless I am mistaken, what does building (without creating) have an effect on?

Reply:
None it seems